### PR TITLE
Style footer

### LINKED ISSUE #4: Footer
- [x] Copy/paste content from the site's footer to the footer document and preview it
- [x] Open an existing page in preview, the footer will be included as a fragment (/footer.plain.html)
- [x]  Style the footer
   see [local dev setup](https://www.hlx.live/developer/tutorial#start-developing-styling-and-functionality)
   see [footer block documentation](https://www.hlx.live/developer/block-collection/footer)

### PR DESCRIPTION
**Pull request by alexcarol**
Fix #4

Test URLs:
- Before: https://main--netcentric--hlxsites.hlx.page/
- After: https://issue-4--netcentric--hlxsites.hlx.page/

### CODE REVIEW

**Comment by alexcarol** [comment]
> I've checked the footer HTML and I saw the SVG icons for social media are inlined in the HTML. Inlining SVGs is generally a good idea if the icons are to be displayed above-the-fold, e.g. in the header.
> 
> Conversely, it's a better idea not to inline SVGs in the footer, to keep the HTML slim, to reduce the number of total DOM elements, and HTML parsing complexity. In addition, I'm proposing to lazy load the footer icons, so they are downloaded only when the footer is about to enter the viewport.
> 
> The easiest way I see, since the SVG icons are not changing color at mouseover, is to use an `img` tag with `loading="lazy"`.
> 
> ```
> twitter
> ```
> 
> Is it possible to make this change? Thanks


We're currently using the franklin [icons](https://www.hlx.live/developer/block-collection/icons) feature. We could consider extending "decorateIcons" to implement it. @mhaack what has your experience been so far, is this something that has been done or discussed in any other projects?

File: blocks/footer/footer.css [hunk 1]
```diff
@@ -1,14 +1,92 @@
...
+.footer a {
+  color: white;
+  text-decoration: none;
+}
+
+/* main footer section that contains the different subsections */
+footer > div > div {
+  display: flex;
+  flex-flow: row wrap;
+  margin: 16px;
```
Comment (joanbeltran) [inline blocks/footer/footer.css:26]: Inner spacings should be padding. the margin overlaps and creates a white line at the bottom

**Comment by mhaack** [comment]
On the icons, I suggest we go with the current approach, did not see issue with that in other projects. If this needs to be optimized we can still do later.

File: blocks/footer/footer.css (hunk 1 above)
Comment (alexcarol) [reply blocks/footer/footer.css]: Fixed, thanks!

**Comment by verlok** [comment]
> On the icons, I suggest we go with the current approach, did not see issue with that in other projects. If this needs to be optimized we can still do later.

I agree.

**Comment by verlok** [comment]
Looks good to me

**Review by jonatan-lledo-netcentric — APPROVED** [review]

File: blocks/footer/footer.css [hunk 2]
```diff
@@ -1,14 +1,92 @@
...
+
+/* social subsection */
+footer > div > div > div:nth-child(3) > ul {
+  display: inline-flex;
+  float: right;
+  padding-inline-start: 0;
+  font-size: large;
+}
+
+.icon {
```
Comment (hannessolo) [inline blocks/footer/footer.css:78]: FYI @alexcarol this affects other blocks too (eg. its breaking the header on my branch after merging). So please always prefix with the block, ie. `footer .icon`.

But don't worry about changing it here, I will do it on the header branch.

**Pull request merged**